signer/src/bitcoin/utxo.rs

```diff
@@ -925,12 +928,21 @@ impl<'a> UnsignedTransaction<'a> {
     }
 }
 
-/// This implementation here includes functions for apportioning fees to a
-/// bitcoin transaction that has already been confirmed. This
-/// implementation is located in this module because it the assumptions for
-/// how the transaction is organized follows the logic in
+/// A trait for figuring out the fees assessed to deposit prevouts and
+/// withdrawal outputs in a bitcoin transaction.
+///
+/// This trait and the default implementations includes functions for
+/// apportioning fees to a bitcoin transaction that has already been
+/// confirmed. This implementation is located in this module because it the
+/// assumptions for how the transaction is organized follows the logic in
```

Comment:
```suggestion
/// confirmed. This implementation is located in this module because the
/// assumptions it makes for how the transaction is organized follows the logic in
```

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/803/commits/84f9fd7bc05073500cd61d46d0a567d77b4f6b18